src/lgtv_rs232/lgtv.py

```diff
@@ -242,7 +242,11 @@ def request(self, command, value):
 
         self._ser.write(request)
 
-        response = str(self._ser.read_until(b'x'), 'ascii').strip('x')
+        response_raw = self._ser.read_until(b'x')
+        # There can be some garbage on console after power on command
+        # Lets filter it out
+        filter(lambda x: x in set(string.printable), response_raw)
```

Comment:
Maybe I'm missing something, but since bytes objects are immutable and the return value is not being captured here, I don't think this line is doing anything. Perhaps changing this to something like `response_filtered = filter(lambda x: ...` would make the `errors='ignore'` argument of the next line unnecessary.

Reply:
(The next line would have to change `response_raw` to `response_filtered` of course.)